webapp/src/main/webapp/iaf/gui/views/common/topnavbar.html

```diff
@@ -42,9 +42,9 @@
                     </li> -->
                 </ul>
             </li>
-            <li ng-if="loggedin">
+            <li ng-if="::userName">{{::userName}}
```

Comment:
I think the if statement should remain, the logout button is not directly related to the user. 
You can display the username here, when available of course :)